packages/calcite-components/src/components/list-item-group/list-item-group.scss

```diff
@@ -1,5 +1,14 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-list-background-color: Specifies the component's background color.
```

Comment:
I’d expect to support “—calcite-list-item-background-color” as well. 

Consider this use case which is representative of how most of our users want to theme: https://community.esri.com/t5/calcite-design-system-questions/change-listitem-background-color-in-code/m-p/1557125#M767

Reply:
We are limiting tokens when shared between grouped components. This is documented in the wiki.

Reply:
How is this different than Accordion Item / Accordion - where we have `-item-background-color`? https://github.com/Esri/calcite-design-system/pull/10181

Reply:
@alisonailea can you comment on the above? Currently, it seems like parent/child tokens are inconsistent.

Reply:
The “item tokens” path seems the easiest to follow both as a maintainer (IMO) and as someone who uses these theming options frequently day to day.

Using the Community post as just one example, it also seems to align with expectations from external folks - theming the element you want to adjust with component-specific nomenclature / rules.

Reply:
Need direction on this one. @alisonailea